docs/csharp/language-reference/operators/equality-operators.md

```diff
@@ -79,10 +87,18 @@ The following example demonstrates the usage of the `!=` operator:
 
 A user-defined type can [overload](operator-overloading.md) the `==` and `!=` operators. If a type overloads one of the two operators, it must also overload the other one.
 
+A record type cannot explicitly overload the `==` and `!=` operators. If you need to change the behavior of the `==` and `!=` operators for record type `T`, implement the <xref:System.IEquatable%601.Equals%2A?displayProperty=nameWithType> method with the following signature:
+
+```csharp
+public virtual bool Equals(T? other);
```

Comment:
If the record type is sealed, `virtual` will cause an error. This is not very important to note though.

Reply:
Again, i think compiler error messages provide enough guidance here.